src/fromJS.js

```diff
@@ -13,12 +14,25 @@ export function fromJS(value, converter) {
   );
 }
 
+function getToSeq(value) {
+  if (Array.isArray(value)) {
+    return IndexedSeq;
+  }
+
+  if (value instanceof Set) {
```

Comment:
To disambiguate, you might want to specifically reference the global Set class

```suggestion
  if (value instanceof global.Set) {
```